.gitignore

```diff
@@ -34,9 +34,9 @@ htmlcov/
 utils/*
 
 # local settings
-.env
+#.env
 .venv
-local.env
+#local.env
```

Comment:
`local.env` should be ignored since it will be our local environment configuration file.